app/models/token.rb

```diff
@@ -0,0 +1,15 @@
+class Token < ApplicationRecord
+  belongs_to :map
+  has_one_attached :image
+
+  validates :name, presence: true
+  validates :image, presence: true
+
+  def scaled_x
```

Comment:
Unless I'm missing it, these two functions aren't used?

Reply:
Ah yes, I was trying to position them correctly in the partial but scrapped it.

Reply:
Removed in c0ebd3a17a66c5acb6b6379a0c849bd28d9329e8